test/test.js

```diff
@@ -13,12 +14,34 @@ async function test(server) {
     // map call occurs in browser context
     /* eslint-disable-next-line no-undef */
     const mapCenter = await page.evaluate(() => map.getCenter());
-    t.assert(mapCenter.lng === 5.9699 && mapCenter.lat === 52.2112, 'map is centered as expected');
+    await t.assert(mapCenter.lng === 5.9699 && mapCenter.lat === 52.2112, 'map is centered as expected');
+    await browser.close();
+    t.end();
+  });
+  await test('correct config is loaded', async (t) => {
+    const browser = await puppeteer.launch({ args: ['--no-sandbox', '--disable-setuid-sandbox'] });
+    const page = await browser.newPage();
+    await page.goto('http://localhost:8123/index.html');
+    /* eslint-disable no-undef */
+    const urlForStandard = await page.evaluate(() => {
+      const layer = nlmaps.leaflet.bgLayer('standaard');
+      /* eslint-disable no-underscore-dangle */
+      /* eslint-disable no-console */
+      return layer._url;
+    });
+    console.log('LAYER URL', urlForStandard);
+    /* eslint-enable no-undef */
+    await t.assert(urlForStandard === 'https://geodata.nationaalgeoregister.nl/tiles/service/wmts/brtachtergrondkaart/EPSG:3857/{z}/{x}/{y}.jpeg', 'url for created layer is as expected');
+    console.log('COMPARE URL', urlForStandard);
+    /* eslint-enable no-console */
+    /* no-underscore-dangle */
+    /* eslint-disable-next-line no-self-compare */
+    await t.assert('hello' === 'hello', 'hello is hello');
```

Comment:
The test looks a bit messy like this. What's the use for asserts like this? Or is this still a debugging state you're in?